src/main/java/edu/kis/powp/jobs2d/drivers/adapter/SpecialDrawerDriver.java

```diff
@@ -0,0 +1,39 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+public class SpecialDrawerDriver extends DrawPanelController implements Job2dDriver {
+    private int startX = 0, startY = 0;
+
+    public SpecialDrawerDriver() {
+        super();
+    }
+
+    @Override
+    public void setPosition(int x, int y) {
+        this.startX = x;
+        this.startY = y;
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        ILine line = LineFactory.getSpecialLine();
```

Comment:
Hardcoded line type (no OCP). What about 3.3.2 test?
Direct coupling with LineFactory class is unnecessary.